stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -209,8 +224,10 @@ internal class PaymentSheetViewModel internal constructor(
     }
 
     fun checkout(checkoutIdentifier: CheckoutIdentifier) {
+        // Clear out any previous errors before setting the new button to get updates.
+        _viewState.value = PaymentSheetViewState.Ready(null)
```

Comment:
Parameter is null by default, so you can omit. Across the PR.

Reply:
I am going to prefer being explicit here.